infra/main.bicep

```diff
@@ -486,6 +486,17 @@ module openAiRoleBackend 'core/security/role.bicep' = if (openAiHost == 'azure')
   }
 }
 
+module openAiRoleSearchService 'core/security/role.bicep' = if (openAiHost == 'azure') {
```

Comment:
This should only be added if they're using the integrated vectorization feature.